files/defaults.yaml

```diff
@@ -0,0 +1,346 @@
+        caPassword: local_ca_password
+      # Distinguished name for Zowe generated certificates.
+      dname:
+        caCommonName: "Zowe Development Instances CA"
+        commonName: "Zowe Development Instances Certificate"
+        orgUnit: "API Mediation Layer"
+        org: "Zowe Sample"
+        locality: "Prague"
+        state: "Prague"
+        country: "CZ"
+      # Validity days for Zowe generated certificates
+      validity: 3650
+
+  
+  # Where to store runtime logs
+  logDirectory: /global/zowe/logs
+
+  
+  # Zowe runtime workspace directory
+  workspaceDirectory: /global/zowe/workspace
+
+  
+  # Where extensions are installed
+  extensionDirectory: /global/zowe/extensions
+
+  configmgr:
+    # STRICT=quit on any error, including missing schema
+    # COMPONENT-COMPAT=if component missing schema, skip it with warning instead of quit
+    validation: "STRICT"
+
+  # >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+  # runtime z/OS job name
+  job:
+    # Zowe JES job name
+    name: ZWE1SV
+    # Prefix of component address space
+    prefix: ZWE1
+
+  # >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+  # This is an ID you use to separate multiple Zowe installs when determining
+  # resource names used in RBAC authorization checks such as dataservices with RBAC
+  # expects this ID in SAF resources 
+  rbacProfileIdentifier: "1"
+
+  # >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+  # This is an ID that can be used by servers that distinguish their cookies from unrelated Zowe installs, 
+  # for purposes such as to allow multiple copies of Zowe to be used within the same client
+  cookieIdentifier: "1"
+
+  # >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+  
+  # You can list your external domains on how you want to access Zowe.
+  # This should be the domain list you would like to put into your web browser's
+  # address bar.
+  externalDomains:
+    # this should be the domain name to access Zowe APIML Gateway
+    - "${{ zos.resolveSymbol('&SYSNAME') }}"
+  
+  # >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+  # This is the port you use to access Zowe Gateway from your web browser.
+  #
+  # In many use cases, this should be same as `components.gateway.port`. But in
+  # some use cases, like containerization, this port could be different.
+  externalPort: 7554
+
+  # >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+  # You can define any Zowe message portions to be checked for and the message added to the 
+  # system log upon its logging, truncated to 126 characters. 
+  sysMessages:
+  #   # Zowe starting
+    - "ZWEL0021I"
+  #   # Zowe started
+    - "ZWEL0018I"
+    - "ZWEL0006I"
+  #   # Zowe ready to use
+    - "ZWES1601I"
+  #   # Zowe stopping
+    - "ZWEL0008I"
+  #   # Zowe stopped
+    - "ZWEL0022I"
+  #   # Zowe components starting
+    - "ZWEL0001I"
+  #   # Zowe components stopped
+    - "ZWEL0002I"
+  #   # API ML components ready
+    - "ZWEAM000I"
+  #   # App server ready
+    - "ZWED0031I"
+  #   # ZSS ready
+    - "ZWES1013I"
```

Comment:
This is creating the validation error `"array uniqueItems"`, when merged with `zowe.yaml`.

Reply:
Oh - I need to remove externalDomains too then.
This is going to need some more testing but is necessary for SMPE to work.

Reply:
I have made this change. `externalDomains` should be fine - it is `sample-domain.com` vs `"${{ zos.resolveSymbol('&SYSNAME') }}"`.

Reply:
yes but if that resolves to a hostname and the user's own yaml specifies the same hostname then it will fail validation